src/cmd/api/main.go

```diff
@@ -29,14 +29,14 @@ func main() {
 		log.Error("Failed to create S3 client: ", err)
 	}
 
-	redisClient := clients.NewRedisClient(cfg.RedisAddr, cfg.RedisPwd, !cfg.DevMode)
-	if err := redisClient.Ping(context.Background()); err != nil {
-		log.Error("Failed to create Redis client: ", err)
+	rabbitmqClient, err := clients.NewRabbitmqClient(cfg.RabbitmqAddr, cfg.RabbitmqUser, cfg.RabbitmqPwd, events.VideoUploaded)
+	if err != nil {
+		log.Error("Failed to create RabbitMQ client: ", err)
```

Comment:
We would rather failed than continue.

You can change the log.Error to log.Fatal